### ISSUE TITLE
[Broken Website] Amazon - Review Your Purchases

### ISSUE THREAD

**Issue by rebeltaz** [darkreader/darkreader#13570]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

When I go to - https://www.amazon.com/review/review-your-purchases/listing - the images have dark backgrounds and are impossible to see:

![Screenshot from 2024-11-30 20-38-58](https://github.com/user-attachments/assets/f97d266d-cd27-4871-ac13-22f63fe6d372)

I can switch from Dynamic to Filter and it looks right:
![Screenshot from 2024-11-30 20-39-18](https://github.com/user-attachments/assets/b0f93d12-a07f-4314-894b-7ac205b4911f)

but I was trying to fix this in the Inversion Fixes Editor. I tried for half an hour, trying different elements, but, as I have no idea what I'm doing... 

Any idea what I need to/do change?

### Website Address

https://www.amazon.com/review/review-your-purchases/listing

### Steps To Reproduce

1. go to link
2. view page
3. shrug

### Operating System

ubuntu linux 22.04

### Web Browser name and version

firefox 132.0.2

### Dark Reader version

4.9.96

**Comment by Myshor** [comment]
I would not be able to fix this as I do not have anything on this list.

If you want to fix it try to find correct selector for the pictures and add it as next one in this part of amazon.* sitefixes:
```
[class*="image-display"],
.a-image-container img,
[class*="imageContainer"] img,
[class*="productImageContainer"] *,
[data-pf=DESKTOP] [class*="img"] * {
    mix-blend-mode: unset !important;
}
```

Eventually you can try to save page as WWW, HTML Only and share it to me (email is available on my github profile page) 😉

**Comment by rebeltaz** [comment]
That was the kick I needed to figure it out. I thought I knew what the tag I was looking for was, I just didn't know where to place it! After seeing your code segment, I managed to add this to that section and got it to work.

`[class*="ryp__review-candidate-new__product-image"],`

![Screenshot from 2024-12-03 20-06-46](https://github.com/user-attachments/assets/b41e7026-2c2c-4971-89c1-0bff4d5e73ff)

I really appreciate your help!
